### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by benjub**
Better reviewed by commit. Commit messages are explanatory.

For the fifth commit, I hope the deprecations are ok with @wlammen and @icecream17 . Note that I had to modify (almost automatically: `MM-PA> expand sb2v` && `MM-PA> min */except sb2v`) a theorem in @wlammen's mathbox and one in @avekens's.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -17744,6 +17748,14 @@ This axiom scheme is logically redundant (see ~ ax12w ) but is used as an
       CNANAFBGNBEOABCHBCINABJKCBLM $.
   $}
 
+  ${
```
Comment (wlammen) [inline set.mm:17751]: No enty in changes-set.txt?

Comment (benjub) [reply set.mm:17751]: Done.

**Review by wlammen — APPROVED** [review]

**Review by avekens — COMMENTED** [review]
Since I am not deep enough in this topic, I cannot approve that the obsolete theorems are really obsolete and can/should be replaced by the indicated ones. Maybe @wlammen can approve this.

Under this assumption, everything is fine, including the change within my mathbox.

**Comment by wlammen** [comment]
Usually, when you have an OLD theorem, you present a new one replacing it.  Here the OLD theorems like stdpc4v became simply specializations of others (stdpc4) without any advantage at all, like reduced axiom usage.  Their existence became pointless, meaningful only in the presence of the old df-sb.  What should a visitor of Metamath learn from it?  At best, if the comment states it this way, that a redefinition can render specialized theorems obsolete.  If it is really of importance, s/he can have a look at the ALT section, where the OLD theorems are still kept in an appropriate context.

**Comment by benjub** [comment]
> Usually, when you have an OLD theorem, you present a new one replacing it. Here the OLD theorems like stdpc4v became simply specializations of others (stdpc4) without any advantage at all, like reduced axiom usage. Their existence became pointless, meaningful only in the presence of the old df-sb. What should a visitor of Metamath learn from it? At best, if the comment states it this way, that a redefinition can render specialized theorems obsolete. If it is really of importance, s/he can have a look at the ALT section, where the OLD theorems are still kept in an appropriate context.

On a related note: as you wrote, it is currently the case that some theorems with a "obsolete as of" comment do not have an "OLD" suffix to their label, which can be confusing to newcomers. Maybe we should simply add an "OLD" suffix to every such theorem ? What do you think ?

**Comment by wlammen** [comment]
I agree

**Comment by benjub** [comment]
> I agree

Done. Can you check ? And I'll stop here for this MR.

**Review by wlammen — APPROVED** [review]

**Pull request merged**